tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -812,6 +814,28 @@ private static uint TryFinallyInner()
         return result;
     }
 
+    private static void TestGenericVirtualMethodCall()
+    {
+        StartTest("Call generic virtual method test");
+
+        TestGenItf implInt = new TestGenItf();
+        implInt.Log<object>(new object());
```

Comment:
I think there is something somewhere that is making this a direct call. I would be really surprised if generic virtual methods actually worked in WASM.

Reply:
Yes, I think you are right, here maybe
https://github.com/dotnet/corert/blob/95dca3191b5f3e446dada07b2cfd24507e5135d0/src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs#L2427-L2429
I think it would be good to keep the test as it was failing previously, so I've renamed it so as to disassociate it from the GVM abbreviation.